src/createPackage.ts

```diff
@@ -1,74 +1,47 @@
-import glob from 'glob';
-import path from 'path';
 import fse from 'fs-extra';
 import { execSync } from 'child_process';
+import copySolidityFiles from './copySolidityFiles';
```

Comment:
For consistency with the test of the imports 😏 
```suggestion
import { copySolidityFiles } from './copySolidityFiles';
```